src/grid/ClientColumn.php

```diff
@@ -74,13 +66,29 @@ public function init()
         return true;
     }
 
+    public function getValue(ActiveRecord $model, User $user): string
+    {
+        if (!isset($model->{$this->nameAttribute})) {
+            return '';
+        }
+        if (
+            $user->can('owner-staff') ||
+            ($user->can('access-reseller') && $model->{$this->idAttribute} && $user->identity->hasOwnSeller($model->{$this->idAttribute})) ||
+            (!empty($model->seller_id) && ($model->seller_id === $user->id || $model->seller_id === $user->identity->seller_id))
+        ) {
+            return Html::a($model->{$this->nameAttribute}, ['@client/view', 'id' => $model->{$this->idAttribute}]);
+        }
```

Comment:
Simplify the condition

```suggestion
        $isStaff = $user->can('owner-staff');
        $isManager = $user->can('access-reseller') && $model->{$this->idAttribute} && $user->identity->hasOwnSeller($model->{$this->idAttribute});
        $isSeller = !empty($model->seller_id) && ($model->seller_id === $user->id || $model->seller_id === $user->identity->seller_id);
        
        if ($isStaff || $isManager || $isSeller) {
            return Html::a($model->{$this->nameAttribute}, ['@client/view', 'id' => $model->{$this->idAttribute}]);
        }
```